src/main/java/org/jitsi/videobridge/DtlsTransport.java

```diff
@@ -300,12 +301,15 @@ private Node createIncomingPipeline()
         dtlsPath.setPredicate(DTLS_PREDICATE);
         PipelineBuilder dtlsPipelineBuilder = new PipelineBuilder();
         dtlsPipelineBuilder.node(dtlsReceiver);
-        dtlsPipelineBuilder.simpleNode(
-                "sctp app packet handler",
-                packetInfo -> {
-                    endpoint.dtlsAppPacketReceived(packetInfo);
-                    return null;
-        });
+        ConsumerNode sctpHandler = new ConsumerNode("sctp app packet handler")
```

Comment:
These two fixes (this one and the one for SRTP below) I think are correct.  Previously, these were implemented as `TransformerNode`s which returned null, so it was returning the buffer to the pool, but functionally they are `ConsumerNode`s.